src/Microsoft.OData.Core/UriParser/Aggregation/ApplyClause.cs

```diff
@@ -51,6 +53,13 @@ public ApplyClause(IList<TransformationNode> transformations)
 
                     break;
                 }
+                else if (transformations[i].Kind == TransformationNodeKind.Compute)
+                {
+                    if (lastAggregateExpressions == null)
+                    {
+                        lastComputeExpressions = (transformations[i] as ComputeTransformationNode).ComputeClause.ComputedItems;

```

Comment:
>lastComputeExpressions [](start = 24, length = 22)

As discussed you may need to keep appending to this collection to cover the case of a series of compute clauses.